static/js/ajax-notifications.js

```diff
@@ -39,7 +42,7 @@ function wsCheckNotifications() {
 
         // Add handling for different kinds of events. For ex: {"event": "notification", "data": {"title": ..}}
         if (msg.event === 'notification') {
-            displayPNotify(msg.data.type, msg.data.title, msg.data.body);
+            displayPNotify(msg.data.type, msg.data.title, msg.data.body, msg.data.hash);
         } else {
             displayPNotify('info', '', msg);
```

Comment:
I added a break-point here but it's never reach this code:

![image](https://user-images.githubusercontent.com/2620870/28018943-83467be4-6555-11e7-9c40-272018ca0231.png)


maybe the websockets @p0psicles ?